upronix.com.domain-setup.json

```diff
@@ -0,0 +1,17 @@
+{
+  "providerId": "upronix.com",
+  "providerName": "Upronix",
+  "serviceId": "domain-setup",
+  "version": 1,
+  "serviceName": "Domain Setup",
+  "logoUrl": "https://upronix.com/domain-connect/logo.png",
+  "description": "Setup an Domain to use Custom Domains",
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "%domain%",
```

Comment:
`%domain%` is a reserved variable, so in the end this would mean creating a subdomain with the same hostname as domain. Like `example.com.example.com`.
I don't think this was the intention.
I guess this template is for subdomains, then right thing to do would be host=@ and `hostRequired` set to `true`. Then the right host would be set with `host` parameter.

Anyway, please test your template here https://pdnsadmin.revproxy.short-lived.de/dc/free/templateedit and provide an appropriate update.